### ISSUE TITLE
upgrade needed linum to new line number mode

### ISSUE THREAD

**Issue by Aneeqasif** [emacs-vs/line-reminder#30]
![image](https://github.com/emacs-vs/line-reminder/assets/78728701/f14ddd44-bc5a-4d0e-9e8d-a03793eb4934)

i am not a emacs expert but it seems like linum mode is obsolete which this package is making use of and enabling this cause unwanted double line in emacs if only way to use this package with default doom emacs is by using indicators which donot loook as visually good as the linum option provide.  i hope u get my point. please adress this issue thanks :)

**Comment by jcs090218** [comment]
The current recommendation is to use `display-line-numbers-mode` (the newer line number mode) plus the `indicators` option. Unfortunately, `display-line-numbers-mode` doesn't allow you to customize the format like `linum-mode` does. 😓

**Comment by Aneeqasif** [comment]
yeah i can understand it is not a big deal though i just wanted the indicator bitmaps to look visually more like they are a one single hunk instead of sperated lines for each line but ig its not possible with indicators so i am happy that i get to use this package thats enough for me .

**Comment by Aneeqasif** [comment]
do u know of any alternatives for this package ?

**Comment by jcs090218** [comment]
> I just wanted the indicator bitmaps to look visually more like they are a one single hunk instead of sperated lines for each line but ig its not possible with indicators so i am happy that i get to use this package thats enough for me .

I'm not quite sure if I understand this. You can design bitmap in any way you want! 😀 

> do u know of any alternatives for this package ?

The closest I could think of is git-gutter or diff-hl.

**Comment by Aneeqasif** [comment]
### i mean with default settings mine look like this
![image](https://github.com/emacs-vs/line-reminder/assets/78728701/79f0a15b-ba2d-4a5f-b724-3be36505eec6)

```lisp
(add-hook 'prog-mode-hook  'line-reminder-mode)
(add-hook 'org-mode-hook 'line-reminder-mode)
(setq line-reminder-bitmap 'line-reminder--default-bitmap)
(setq line-reminder-thumbnail t)
(setq line-reminder-show-option 'indicators)
```


### while yours look like this 
![image](https://github.com/emacs-vs/line-reminder/assets/78728701/bed3f97e-ca9a-4665-8923-aacc76ce0cef)


how can i make the bitmaps to look like its a part of same line as you have shown in screen shot i have tried it with jetbrainsmono and iosevka fonts any clue?

**Comment by jcs090218** [comment]
Try

```elisp
(fringe-helper-define 'my-bitmap nil
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.." "..xxx.."
  "..xxx.." "..xxx..")

(setq line-reminder-bitmap 'my-bitmap)
```
